js/script.js

```diff
@@ -91,6 +91,7 @@ var script = {
 
 	"h Go ahead and create an amazing Game!",
 	"h I can't wait to see what story you'll tell!",
+	"pee in my mouth",
 	"end"
```

Comment:
Wakarimashita
On May 28, 2016 1:19 AM, "Anthony Z" <notifications@github.com> wrote:

> In js/script.js
> <https://github.com/Tamakimouto/ah-now-i-see/pull/3#discussion_r64984300>:
>
> > @@ -91,6 +91,7 @@ var script = {
> >
> >  	"h Go ahead and create an amazing Game!",
> >  	"h I can't wait to see what story you'll tell!",
> > +	"pee in my mouth",
> >  	"end"
>
> You should really not add this into a commit you do. It'll be there
> forever in the commit history if I were to merge it in.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/Tamakimouto/ah-now-i-see/pull/3/files/638bc2a0d968a612accb5ad588ec9f011c690b5b#r64984300>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AG2ycSB6qxa6lPrwifL95sTu18aELYulks5qF9BEgaJpZM4IkNAb>
> .
>